pkg/cmd/const.go

```diff
@@ -0,0 +1,15 @@
+package cmd
+
+import (
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+)
+
+var TRUE string = "true"
+var VeleroNameSpace string = "velero-test"
+var CaptureFlag string = "BE_EXECUTE"
```

Comment:
Since these two variables `VeleroNameSpace` and `CaptureFlag` are used by UT only, can we move these definitions into a testing file?

Reply:
Yes, agree that.